common/dbc.py

```diff
@@ -15,7 +15,10 @@ def int_or_float(s):
 
 class dbc(object):
   def __init__(self, fn):
-    self.txt = open(fn).read().split("\n")
+
+    with open(fn, 'r') as datafile:
+      self.txt = map(str.split, datafile.read().split("\n"))
```

Comment:
That is a blunder on my part, I should have mapped str.strip. Whether that would speed up or slow down the module I am now questioning, along with many other things.